app/data_models/session_data.py

```diff
@@ -1,13 +1,14 @@
+# pylint: disable=unused-argument
 from typing import Any, Optional
 
 
 class SessionData:
     def __init__(
         self,
+        language_code: Optional[str],
         tx_id: Optional[str],
         schema_name: Optional[str],
         period_str: Optional[str],
```

Comment:
Any reason for the order change?. Worth also enforcing keyword args. I think we always pass as keyword args already but just not enforced. If it was, this order change wouldn't have mattered.

Reply:
Enforced keyword args and removed the extra keys